gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -287,7 +287,7 @@ def configure_interfaces(self, context, resource_data):
         """
 
         mgmt_ip = resource_data['mgmt_ip']
-
+        import time;time.sleep(15)
```

Comment:
smaller intervals because what if push gets succeed in the next retry which means it would save us 14secs...